### PR TITLE
created code_of_conduct.md file

### PR DESCRIPTION
**Pull request by Sloth-Panda**
## Issue that this pull request solves

 Closes:  #75 

### Brief description of what is fixed or changed
Made code_of_conduct.md  file

## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] My changes does not break the current system and it passes all the current test cases.

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface

## Other information

Any other information that is important to this pull request

### CODE REVIEW

**Comment by Sloth-Panda** [comment]
Hey, Let me know if any changes are required

File: code_of_conduct.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+when an individual is representing the project or its community.
+
+###
+
+## Enforcement
+
+---
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting a project maintainer at [INSERT EMAIL ADDRESS]. All
```
Comment (Kajol-Kumari) [inline code_of_conduct.md:70]: please replace `[INSERT EMAIL ADDRESS]` by `hitktechcommunity@gmail.com`

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@Sloth-Panda please make this minor change and it will be good to go :)

**Comment by Sloth-Panda** [comment]
Hey, can you check this out now?

**Review by Kajol-Kumari — APPROVED** [review]
LGTM :+1:

**Comment by Kajol-Kumari** [comment]
@all-contributors please add @Sloth-Panda  for doc

**Comment by Sloth-Panda** [comment]
@Kajol-Kumari is not this pr counted for swoc?

**Pull request merged**